api/helpers/indexes.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
```

Comment:
indexes.js might be a bad name for this... really close to index.js (which is kind of a reserved word) maybe text-index.js or something like that would have been clearer.  Although what you did here was ok, just thinking the future yourself or someone else.

Reply:
I see your point here, node does treat "index.js" as its default and this may cause some confusion. I'll rename in a future PR. Cheers